Outdated example in 'Migrating to module Workers'

### Which Cloudflare product does this pertain to?

Workers

### Existing documentation URL(s)

https://developers.cloudflare.com/workers/learning/migrating-to-module-workers/#configuring-your-wranglertoml

### Section that requires update

Configuring your wrangler.toml

### What needs to change?

[build.upload]
format = "modules"
dir = "./src"
main = "./worker.js" # becomes "./src/worker.js"

[[build.upload.rules]]
type = "ESModule"
globs = ["**/*.js"]

### How should it change?

When you run Wrangler with this configuration you've got warnings:

    - Deprecation: "build.upload.format":
      The format is inferred automatically from the code.
      
    - Deprecation: "build.upload.dir":
      Use the top level "main" field or a command-line argument to specify the entry-point for the Worker.
      
    - Deprecation: The `build.upload.rules` config field is no longer used, the rules should be specified via the `rules` config field. Delete the `build.upload` field from the configuration file, and add this:
      ```
      [[rules]]
      type = "ESModule"
      ```


### Additional information

In my case (using TypeScript) I also got an error:
 [ERROR] [plugin wrangler-module-collector] Cannot read properties of undefined (reading 'map')

--- a/content/workers/learning/migrating-to-module-workers.md
+++ b/content/workers/learning/migrating-to-module-workers.md
@@ -122,15 +122,7 @@ To add support for module Workers to an existing project, update your `wrangler.
 name = "my-worker"
 type = "javascript"
 workers_dev = true
-
-[build.upload]
-format = "modules"
-dir = "./src"
-main = "./worker.js" # becomes "./src/worker.js"
-
-[[build.upload.rules]]
-type = "ESModule"
-globs = ["**/*.js"]
+main = "src/index.js"
 
 # Uncomment if you have a build script.
 # [build]

